qemu/tests/balloon_check.py

```diff
@@ -24,12 +24,14 @@ def __init__(self, test, params, env):
 
         self.vm = env.get_vm(params["main_vm"])
         self.session = self.get_session(self.vm)
+        self.env["memory_stable"] = 0
```

Comment:
if it's better to use Ture/False

Reply:
Should be okay, other env flag also use this kind of usage,such as:
self.env["balloon_test"] = 0